### ISSUE TITLE
Invalid XPath error for Find...menuItemWithIndex

### ISSUE THREAD

**Issue by pbrosgol-aurotfp** [loyada/dollarx#17]
**Scenario**
Select an item from a dropdown menu and then, to avoid a race condition, ensure the menu has closed.

**Example code**
```
    clickOn(somePage.menuItemWithIndex(1));
    find(body.that(ElementProperties.not(contains(somePage.menuItemWithIndex(1)))));
````
**Observed error**
Caused by: org.openqa.selenium.InvalidSelectorException: invalid selector: Unable to locate an element with the xpath expression //body[not(descendant::(//li[@role='option'])[2])] because of the following error:
SyntaxError: Failed to execute 'evaluate' on 'Document': The string '//body[not(descendant::(//li[@role='option'])[2])]' is not a valid XPath expression.

**Comment by loyada** [comment]
Looks like there is no way to define this expression in xpath. Will have to add logic to block the creation of such a Path

**Comment by loyada** [comment]
Fixed in version 0.38
